test/reduce/memory_table.wast

```diff
@@ -27,5 +30,10 @@
    (call_indirect (type $i) (i32.const 0))
   )
  )
+ (func $f5 (result i32)
+  (i32.add
+   (i31.get_s (table.get 1 (i32.const 0)))
+   (i31.get_u (table.get 1 (i32.const 1)))
```

Comment:
With the suggestion above I think one of these two operations will get reduced but not the other.